client/my-sites/stats/stats-subscribers-overview/index.tsx

```diff
@@ -22,64 +22,71 @@ function calculateQueryDate( daysToSubtract: number ) {
 }
 
 // calculate the stats to display in the cards
-function SubscribersOverviewCardStats( subscribersData: SubscribersData[] ) {
-	const getCount = ( index: number ) => {
-		return subscribersData[ index ]?.data[ 0 ]?.subscribers || 0;
+function SubscribersOverviewCardStats( subscribersData: SubscribersData, index: number ) {
+	const getCount = () => {
+		return subscribersData?.data?.[ 0 ]?.subscribers || 0;
 	};
 
-	const overviewCardStats = [
-		{
-			heading: translate( 'Today' ),
-			count: getCount( 0 ),
-		},
-		{
-			heading: translate( '30 days ago' ),
-			count: getCount( 1 ),
-		},
-		{
-			heading: translate( '60 days ago' ),
-			count: getCount( 2 ),
-		},
-		{
-			heading: translate( '90 days ago' ),
-			count: getCount( 3 ),
-		},
-	];
+	let heading;
+	switch ( index ) {
+		case 0:
+			heading = translate( 'Today' );
+			break;
+		case 1:
+			heading = translate( '30 days ago' );
+			break;
+		case 2:
+			heading = translate( '60 days ago' );
+			break;
+		case 3:
+			heading = translate( '90 days ago' );
+			break;
+		default:
+			heading = '';
+			break;
+	}
 
-	return overviewCardStats;
+	const overviewCardStat = {
+		heading,
+		count: getCount(),
+	};
+
+	return overviewCardStat;
```

Comment:
```suggestion
	return {
		heading,
		count: subscribersData?.data?.[ 0 ]?.subscribers || 0,
	};
```

I didn't see an obvious benefit to declaring a `getCount` function, so I inlined it here.